contacts.module

```diff
@@ -253,7 +275,35 @@ function contacts_dashboard_block_edit_link_pre_render(array $element) {
  */
 function contacts_entity_type_build(array &$entity_types) {
   /* @var $entity_types \Drupal\Core\Entity\EntityTypeInterface[] */
-  $entity_types['profile']->setFormClass('crm_dashboard', ContactsProfileForm::class);
+  $entity_types['profile']->setFormClass('default', ContactsProfileForm::class);
+  $entity_types['profile']->setFormClass('contacts_dashboard', ContactsProfileForm::class);
+  $entity_types['profile']->setFormClass('user_dashboard', ContactsProfileForm::class);
+}
+
+/**
+ * Implements hook_ENTITY_TYPE_insert() for profile_type.
+ *
+ * Set up entity displays for contacts and user dashboards.
+ */
+function contacts_profile_type_insert($entity) {
```

Comment:
Do we want to assume custom displays for every profile type, or should we assume default and let site builders opt into custom?

Reply:
Updated.